curriculum/challenges/english/blocks/lab-customer-complaint-form/67279fe50237291f80eed8b8.md

```diff
@@ -214,26 +214,26 @@ document.getElementById("complaint-description").value = "Less than 20 chars";
 assert.isFalse(validateForm()["complaint-description"]);
 ```
 
-When `#other-complaint` is checked and the value of `#complaint-description` is changed to a valid value, you should set its border color to `green`.
+When `#other-complaint` is checked and the value of `#complaint-description` is changed to a valid value, you should set the parent fieldset's border color to `green`.
```

Comment:
this contradicts the user stories, if you need to make such a change for the tests to work, it can't be update here only